lib/facebook.ex

```diff
@@ -616,6 +712,32 @@ defmodule Facebook do
     |> ResponseFormatter.format_response()
   end
 
+  @doc """
+  Decodes a signed request from a client SDK (in-app payments), verifies the
+  signature and -if valid- returns its contents.
```

Comment:
why the dashes around `if valid`? IIRC dashes don't do any special kind of formatting, right?

Reply:
These should have been brackets, I changed the sentence a bit to make it more understandable.